tests/unit_tests/blas/batch/gemm_batch_stride.cpp

```diff
@@ -99,19 +99,28 @@ int test(device *dev, oneapi::mkl::layout layout, int64_t batch_size) {
         default: break;
     }
 
-    vector<fp, allocator_helper<fp, 64>> A(stride_a * batch_size), B(stride_b * batch_size);
-    vector<fp, allocator_helper<fp, 64>> C(stride_c * batch_size), C_ref(stride_c * batch_size);
+    vector<Ta, allocator_helper<Ta, 64>> A(stride_a * batch_size);
+    vector<Ta, allocator_helper<Tb, 64>> B(stride_b * batch_size);
+    vector<Tc, allocator_helper<Tc, 64>> C(stride_c * batch_size),
+        C_cast_ref(stride_c * batch_size);
+    vector<Ts, allocator_helper<Ts, 64>> A_ref(stride_a * batch_size), B_ref(stride_b * batch_size),
+        C_ref(stride_c * batch_size);
 
     for (i = 0; i < batch_size; i++) {
         rand_matrix(A.data() + stride_a * i, layout, transa, m, k, lda);
         rand_matrix(B.data() + stride_b * i, layout, transb, k, n, ldb);
         rand_matrix(C.data() + stride_c * i, layout, oneapi::mkl::transpose::nontrans, m, n, ldc);
     }
 
-    C_ref = C;
+    for (size_t i = 0; i < A.size(); ++i)
+        A_ref[i] = A[i];
+    for (size_t i = 0; i < B.size(); ++i)
+        B_ref[i] = B[i];
+    for (size_t i = 0; i < C.size(); ++i)
+        C_ref[i] = C[i];
```

Comment:
I know that the existing code doesn't always conform to it, but there is a [style guide](https://github.com/oneapi-src/oneMKL/blob/develop/CONTRIBUTING.md#coding-style). There are a few places where it calls for putting `{}` around the body of your for loops.

Reply:
Iv placed `{}` where I think the style guide recommends, let me know if you see a place iv missed